src/scenes/gameuiscreen.ts

```diff
@@ -45,23 +46,30 @@ export class GameUiScreen extends Phaser.Scene {
     public displayDrawnTiles(): void {
         assert(this.drawnTilesContainer);
         this.drawnTilesContainer.removeAll(true);
+
         const myTiles: Nullable<Tile[]> = TileManager.getInHand();
         assert(myTiles && this.uiBackground);
 
         const nrTiles: int = myTiles.length;
         const tileSpacing: int =
             (this.uiBackground.width - nrTiles * 128) / (nrTiles + 1);
+        const myTurn: boolean = PlayerManager.checkMyTurn();
 
         for (let i: int = 0; i < nrTiles; i++) {
-            this.drawnTilesContainer.add(
-                this.add.image(
-                    this.uiBackground.getTopLeft().x +
-                        (tileSpacing + 128 / 2) +
-                        i * (128 + tileSpacing),
-                    ScreenHeight - 100,
-                    `tile${myTiles[i].type}`,
-                ),
+            const image: Phaser.GameObjects.Image = this.add.image(
+                this.uiBackground.getTopLeft().x +
+                    (tileSpacing + 128 / 2) +
+                    i * (128 + tileSpacing),
+                ScreenHeight - 100,
+                `tile${myTiles[i].type}`,
             );
+            if (myTurn) {
+                this.drawnTilesContainer.add(image);
+            } else {
+                this.drawnTilesContainer.add(
+                    image.setAlpha(0.5).setTint(0xfefeee),
+                );
+            }
```

Comment:
would do it like this:
```
if (!myTurn) {
    image.setAlpha(0.5);
    image.setTint(0xfefeee);
}
this.drawnTilesContainer.add(image);
```

Reply:
Ah and also you could disable the interactivity while its not myTurn. (so the player cant drag it to place it)

Reply:
https://newdocs.phaser.io/docs/3.54.0/focus/Phaser.GameObjects.GameObject-disableInteractive#:~:text=An%20object%20that%20is%20disabled,Object%20then%20use%20removeInteractive%20instead.